scripts/install-multi-user.sh

```diff
@@ -701,7 +701,10 @@ install_from_extracted_nix() {
         cd "$EXTRACTED_NIX_PATH"
 
         _sudo "to copy the basic Nix files to the new store at $NIX_ROOT/store" \
-              rsync -rlpt --chmod=-w ./store/* "$NIX_ROOT/store/"
+              cp -rdp ./store/* "$NIX_ROOT/store/"
+
+        _sudo "to make the new store non-writable at $NIX_ROOT/store" \
+              chmod ugo-w "$NIX_ROOT/store/"
```

Comment:
```suggestion
              chmod -w "$NIX_ROOT/store/"
```

Reply:
Good point about `-R`!

removing `ugo` breaks the installer as write permission is left for the group.

Reply:
Doesn't -w remove it for all? I would have thought it bad whatever.

Reply:
I thought that too, but it removes it only for the user.